nero_controller.py

```diff
@@ -0,0 +1,42 @@
+
+from raspberry_model import RaspberryModel
+from mock_model import MockModel
+from nero_view import NeroView
+from typing import Optional
+import platform
+
+
+class NeroController:
```

Comment:
Might be worth it to have a controller per view, since you're probably going to need different data/perform a different analysis per view. Prob where you're going but wanted to write it down

Reply:
And then a big controller that handles switching between views, recognizing button presses, etc

Reply:
i disagree, that defeats the purpose of a controller in MVC architecture